README.md

```diff
@@ -18,6 +18,39 @@ To publish a `production` tag of the scraper image, sync the `main` branch with
 git push origin main:deploy
 ```
 
+### Testing locally with councilmatic
```

Comment:
Nice.

Reply:
Since this is addressed in the docs website, how about striking from the README?